src/backend/PhotoAPI/API/Controllers/PhotosController.cs

```diff
@@ -31,6 +31,15 @@ public Task<IEnumerable<PhotoListDTO>> GetCurrentUserPhotos()
             return _photoService.GetPhotosAsync(userId);
         }
 
+        [HttpGet("{photoId}")]
+        public async Task<ActionResult<PhotoViewDTO>> GetCurrentUserPhoto(System.Guid photoId)
+        {
+            PhotoViewDTO photo = await _photoService.GetPhotoOrDefaultAsync(photoId);
+
+            if (photo == null) return NotFound();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/iamprovidence/Lama/pullRequest?prid=4949566)